packages/server/api/src/app/authentication/authentication-service/index.ts

```diff
@@ -24,12 +29,18 @@ export const authenticationService = {
     await hooks.get().preSignUp(params);
     const user = await createUser(params);
 
-    const { token, refresh_token } = await openopsTables.createUser({
-      name: `${params.firstName} ${params.lastName}`,
-      email: params.email,
-      password: params.password,
-      authenticate: true,
-    });
+    // The default superuser should already exist, since the tables container
+    // creates it on startup, and update its password if it was changed.
+    const isSuperuser =
+      params.email == system.getOrThrow(AppSystemProp.OPENOPS_ADMIN_EMAIL);
+    const { token, refresh_token } = isSuperuser
+      ? await authenticateUserInOpenOpsTables(params.email, params.password)
```

Comment:
I don't understand this logic.

This func will be called in 2 instances:
1. Either it's brand new installation and it's the first time we're creating a seed user at all.
2. We're registering with a non-admin user

For 1, why would we need authenticateUserInOpenOpsTables?

Reply:
The admin user is unique, as its parallel in the tables container is created on the tables container boot, and not during the app user sign up.

Reply:
The signUp includes a postSignUp hook that creates an organization and a tables workspace, and adds the admin to them. The original code used the signUp function - didn't see a reason to change that (will require refactoring of the postSignUp function).